features.json

```diff
@@ -12,6 +12,7 @@
     "ember-container-inject-owner": true,
     "ember-htmlbars-local-lookup": null,
     "ember-application-engines": null,
-    "ember-glimmer": null
+    "ember-glimmer": null,
+    "ember-runtime-computed-uniq-by": true
```

Comment:
Should be `null`. @rwjblue will make it `true` after this is given a "go" by core (likely on Friday if we land this)